layouts/basic/modules/PriceBooks/Modals/SpecifyMargin.tpl

```diff
@@ -0,0 +1,18 @@
+{*<!-- {[The file is published on the basis of YetiForce Public License 3.0 that can be found in the following directory: licenses/LicenseEN.txt or yetiforce.com]} -->*}
+{strip}
+	<div class="tpl-PriceBooks-Modals-SpecifyMargin js-modal-body" data-js="container">
+		<div class="col-12 row m-0">
+			<div class="fieldLabel col-sm-4 col-xl-4 pl-0 text-lg-left text-xl-right u-text-ellipsis">
+				<label class="text-right muted small font-weight-bold">
+					{\App\Language::translate('LBL_MARGINP')}
+				</label>
+			</div>
+			<div class="fieldValue col-sm-8 col-xl-8 px-0 px-sm-1">
+					<div class="input-group input-group-sm">
+							<input type="text" class="smallInputBox form-control form-control-sm js-margin"
+									data-validation-engine="validate[required,funcCall[Vtiger_NumberUserFormat_Validator_Js.invokeValidation]]" />
```

Comment:
data-validation-engine="validate[required,funcCall[Vtiger_NumberUserFormat_Validator_Js.invokeValidation]]" ? in js or in tpl?